packages/create-sitecore-jss/src/initializers/nextjs-styleguide/index.ts

```diff
@@ -1,30 +1,34 @@
 import chalk from 'chalk';
 import path from 'path';
+import { prompt } from 'inquirer';
 import { ParsedArgs } from 'minimist';
 import { Initializer } from '../../common/Initializer';
-import { Answer } from '../../common/Answer';
+import { userPrompts } from './user-prompts';
 import { isJssApp, openPackageJson } from '../../common/utils/helpers';
 import { transform } from '../../common/steps/index';
+import { NextjsStyleguideAnswer } from './NextjsStyleguideAnswer';
 
 export class NextjsStyleguideInitializer implements Initializer {
   async init(args: ParsedArgs) {
-    // set destination to cwd, or read from args
-    args.destination = args.destination || path.resolve(process.cwd());
-    args.post = true;
-
     let pkg;
-    // read package.json in target destination, check if app is JSS app
+
     if (!args.yes) {
       pkg = openPackageJson(`${args.destination}\\package.json`);
       isJssApp('nextjs-styleguide', pkg);
     }
 
-    // derive variables from package.json
-    // read the package.json to get the appName
-    args.appName = args.appName || pkg?.config.appName || 'default';
-    args.appPrefix = args.appPrefix || pkg?.config?.prefix || false;
+    const answers: NextjsStyleguideAnswer = {
+      destination: args.destination || path.resolve(process.cwd()),
```

Comment:
Aren't we guaranteed `args.destination` is populated? Else the `openPackageJson` call above would be incorrect.

Reply:
Correct, I reused `argv.destination`